framework/src/multiapps/MultiApp.C

```diff
@@ -124,9 +124,6 @@ MultiApp::MultiApp(const InputParameters & parameters):
 
   mooseAssert(_input_files.size() == 1 || _positions.size() == _input_files.size(), "Number of positions and input files are not the same!");
 
-  // Fill in the _positions vector
-  fillPositions();
```

Comment:
This looks suspicious.  Do we not need this at all? I don't see a corresponding re-add of this removed line.